pkg/component/networking/istio/test_charts/ingress_autoscaler.yaml

```diff
@@ -39,4 +39,4 @@ spec:
       policies:
       - type: Pods
         value: 1
-        periodSeconds: 60
\ No newline at end of file
+        periodSeconds: 60
```

Comment:
The `registry` adds a line break automatically at the end of every resource when `SerializedObjects()` is called (see [here](https://github.com/gardener/gardener/blob/4267f77d8eecb07971b902bec85920f705e8e46b/pkg/utils/managedresources/registry.go#L158)). This is necessary to form a valid output, because otherwise you would have something like 

```
kind: foo
fieldA: valueA
fieldB: valueB---
kind: bar
```

There will be a string mismatch if you don't add line breaks to the test manifests. I would argue that it's best practice to have such line breaks which is why I added them to these manifests.